config/redirects.yml

```diff
@@ -296,6 +296,7 @@ redirects:
   "/funding-your-training": "/funding-and-support"
   "/tips-on-applying-for-teacher-training": "/train-to-be-a-teacher/how-to-apply-for-teacher-training"
   "/get-support-training-to-teach-if-you-are-disabled": "/funding-and-support/if-youre-disabled"
+  "/assessment-only-providers": "/train-to-be-a-teacher/assessment-only-route-to-qts"
```

Comment:
@EllieNodder Need to add in another redirect from /train-to-be-a-teacher/if-you-have-experience to the new page as that goes to a 404 at the moment

Reply:
Also - don't you need to delete the /assessment-only-providers page as well? Looks like it is just redirected rather than deleted